tests/test_migrations.py

```
@@ -0,0 +1,106 @@
+        from NerdyPy.bot import NerpyBot
+
+        await NerpyBot.on_ready(mock_self)
+
+    assert sentinel.exists(), "on_ready must write /tmp/nerpybot_ready"
+
+
+def test_run_migrations_locates_alembic_ini():
+    """run_migrations() should locate alembic.ini relative to bot.py."""
+    with (
+        patch("NerdyPy.bot.Config") as mock_cfg_cls,
+        patch("NerdyPy.bot.alembic_command") as mock_cmd,
```

Flake8 (7.3.0)

[error] local variable 'mock_cmd' is assigned to but never used

(F841)